third_party/mesa/src/freedreno/ir3/instr-a3xx.h

```diff
@@ -24,16 +24,33 @@
 #ifndef INSTR_A3XX_H_
 #define INSTR_A3XX_H_
 
+// GOOGLE: A pragma pack directive is needed to pack a structure in Windows.
+#if defined(__GNUC__)
 #define PACKED __attribute__((__packed__))
+#elif defined(_MSC_VER)
+#define PACKED 
```

Comment:
hmm, i may not get this. 
this means we define `PACKED` as empty, and then add 2 `pragma`
#pragma pack(push, 1)
#pragma pack(pop)

But if we want the pragma to affect a struct, we need to wrap the struct declaration with the pragma. Do i miss anything here?

(whereas with `__attribute__((__packed__))`, we make sure the struct is packed.)

Reply:
Correct, this is in the wrong spot since the nothing is packed here. I moved the PACKED msvc definition to the other instr-a3xx file and moved the pragma pack(pop) to be after the last packed struct